plugin/tests/unit/sdk_v2/plugin/test_test_plugin_v2_smoke.py

```
@@ -0,0 +1,378 @@
+            return None
+
+        def subscribe(self, on="add"):
+            def _decorator(fn):
+                return fn
+
+            return _decorator
+
+    watcher = _RaceWatcher()
+
+    class _RaceBusList(_BusList):
+        def watch(self, ctx, *, bus=None, debounce_ms=None):
```

Ruff (0.15.5)

[warning] Unused method argument: `ctx`

(ARG002)